tests/zenko_tests/node_tests/ui/cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "video": false
+}
```

Comment:
Need to refresh my mind a little bit on how zenko ci is setup on 2.x

but setting this option to `true` and finding a way to upload the videos from Cypress testsuite could be useful.